container/appmaster/Dockerfile

```diff
@@ -1,19 +1,45 @@
 # Includes libmesos.so
 FROM mesosphere/mesos:1.0.1-2.0.93.ubuntu1404
 
+RUN apt-get update && \
+    apt-get install -y \
+            runit \
+            curl \
+            krb5-user
+
 # The base image contains java 7, but it has no environment variables set for it.
 ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64/jre
 
+ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
+
+ENV HADOOP_CONF_DIR /etc/hadoop/conf/
+
+ADD runit/service /var/lib/runit/service
+
+ADD runit/init.sh /sbin/init.sh
+
+RUN chmod +x /sbin/init.sh
+
+RUN ln -s /var/lib/runit/service/flink /etc/service/flink
+
+RUN chmod -R ugo+rw /etc/service
+
+RUN chmod -R ugo+rw /var/lib/
+
+RUN chmod -R ugo+rw /var/run/
+
+RUN chmod -R ugo+rw /var/log/
+
 WORKDIR /
 
 # Copy custom build to image.
-COPY flink/flink-dist/target/flink-1.2-SNAPSHOT-bin/ .
+COPY flink/flink-dist/target/flink-1.3-SNAPSHOT-bin/ .
```

Comment:
I have bumped the Flink sub-project commit since the Flink codebase from 1.2 snapshot will fail if token cache is not available when the cluster is started. It was fixed in the most recent patch which is available in Flink-1.3-snapshot.